src/main/java/io/github/moderngamingworlds_mods/woodenutilities/common/recipes/WoodcutterRecipe.java

```diff
@@ -0,0 +1,147 @@
+    public static final String SERIALIZER_ID = "woodcutter_serializer";
+
+    public ResourceLocation recipeLoc;
+
+    public final String group = WoodenUtilities.getInstance().modid.concat(":").concat(SERIALIZER_ID);
+    public List<String> requiredMods;
+    public Ingredient ingredient;
+    public ItemStack result;
+
+    public WoodcutterRecipe(ResourceLocation recipeLoc, Ingredient ingredient, ItemStack result, List<String> requiredMods) {
+        this.recipeLoc = recipeLoc;
+        this.ingredient = ingredient;
+        this.result = result;
+        this.requiredMods = requiredMods;
+    }
+
+    public WoodcutterRecipe(Ingredient ingredient, ItemStack stack) {
+        this(new ResourceLocation(WoodenUtilities.getInstance().modid, stack.getItem().getRegistryName().getPath() + "woodcutter"),
+                ingredient,
+                stack,
+                List.of("minecraft"));
+    }
+
+    @Override
+    public boolean matches(Container container, @Nonnull Level level) {
+        return this.ingredient.test(container.getItem(0));
+    }
+
+    @Override
+    public ItemStack assemble(Container container) {
+        return this.result.copy();
+    }
+
+    @Override
+    public boolean canCraftInDimensions(int w, int h) {
+        return true;
+    }
+
+    @Override
+    public ItemStack getResultItem() {
+        return this.result;
+    }
+
+    @Override
+    public ItemStack getToastSymbol() {
+        return new ItemStack(ModBlocks.woodCutter);
+    }
+
+    @Override
+    public ResourceLocation getId() {
+        return this.recipeLoc;
+    }
+
+    @Override
+    public RecipeSerializer<?> getSerializer() {
+        return ModRecipes.WOODCUTTER_SERIALIZER;
+    }
+
+    @Override
+    public RecipeType<?> getType() {
+        return ModRecipes.WOODCUTTER;
+    }
+
+    public static class WoodcutterSerializer extends ForgeRegistryEntry<RecipeSerializer<?>> implements RecipeSerializer<WoodcutterRecipe> {
+
+        @Override
+        public WoodcutterRecipe fromJson(ResourceLocation recipeLoc, JsonObject recipeJson) {
+
+            var requireMods = List.of("minecraft");
+
+            if (GsonHelper.isValidNode(recipeJson, "requireMods")) {
+                requireMods = StreamSupport.stream(GsonHelper.getAsJsonArray(recipeJson, "requireMods").spliterator(), false).
+                        map(JsonElement::getAsString).collect(Collectors.toList());
+            }
+
+            Ingredient ingredient;
+            if (GsonHelper.isArrayNode(recipeJson, "ingredient")) {
+                ingredient = Ingredient.fromJson(GsonHelper.getAsJsonArray(recipeJson, "ingredient"));
+            } else {
+                ingredient = Ingredient.fromJson(GsonHelper.getAsJsonObject(recipeJson, "ingredient"));
+            }
+
+            var resultItem = GsonHelper.getAsString(recipeJson, "result");
+            int count = GsonHelper.getAsInt(recipeJson, "count");
```

Comment:
`count` should be optional, default 1.

Reply:
this count is the result amount

Reply:
Yes, it is. And its default should be one. If you don't set it, it should always be one. Now, if you don't set it, the recipe is incorrect. Default to one would make recipes simpler.